style.css

```diff
@@ -0,0 +1,114 @@
+body{
+  width: 100%;
+  max-width: 2180px;
+  margin: 0 auto;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  font-size:  18px;
+  font-weight: 400;
+  line-height: 150%;
+  text-decoration: none;
+  background-color: #274001;
+  color: #BF9B6F;
+  display: flex;
+  flex-direction: column;
+  padding: 0;
+  }
+  * {
+    box-sizing: border-box;
+  }
+  .header {
+    font-size: 18px;
+    flex-direction: row;
+    align-items: flex-end;
+    background-color: #8C3604;
+    flex-grow: 1;
+    flex-wrap: wrap;
+  }
+  .main {
+    display: flex;
+    align-items: center;
+    margin-top: 10px;
+  }
+  .footer {
+    padding: 20px;
+    background-color: #8C3604;
+    color: #BF9B6F;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    height: 1%;
+  }
+  .container {
+    display: flex;
+    flex-direction: column;
+    margin: 0 auto;
+    justify-content: space-between;
+    margin-left: 1%;
+    max-width: 2180px;
+    min-height: 50px;
+  }
+  .main-container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+  }
+  .footer-container {
+    flex-direction: row;
+    justify-content: center;
+    font-size: 16px;
+  }
+  .nav-list {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    list-style: none;
+  }
+  .nav-item {
+    margin: 10px;
+    align-items: center;
+  }
+  .nav-link {
+    text-decoration: none;
+    color: #ffffff;
+    transition: .3s;
+  }
+  .section {
+    width: calc(33%-20px);
+    padding: 20px;
+    padding-bottom: 40px;
+  }
```

Comment:
Старайся избегать вычисления в ширине ибо потом не понятно от чего и как ты считал 
да еще он у тебя и не сработал 
![image](https://github.com/wladyslawph/rsschool-cv/assets/87061061/c2e120a3-cfd4-4151-bab9-074f86758b3c)

```suggestion
  .section {
    width: 100%;
    padding: 20px;
    padding-bottom: 40px;
  }
```

Reply:
Если задать отступы хотел секциям то можно было добавить margin